modules/extensions/src/mask/mask.ts

```diff
@@ -31,6 +32,10 @@ export default class MaskExtension extends LayerExtension {
   static defaultProps = defaultProps;
   static extensionName = 'MaskExtension';
 
+  initializeState(this: Layer<MaskExtensionProps>) {
+    this.context.deck?._addDefaultEffect(new MaskEffect());
```

Comment:
As above, do we want to let this silently succeed if there is no deck instance?

Reply:
There is no deck instance when running test utilities.